Makefile

```diff
@@ -171,20 +168,13 @@ $(Worker_Enclave_u_Object): service/Enclave_u.o
 	$(AR) rcsD $@ $^
 	cp $(Worker_Enclave_u_Object) ./lib
 
-$(Worker_Name): $(Worker_Enclave_u_Object) $(Worker_SRC_Files)
+$(Worker_Name): $(Worker_Enclave_u_Object) $(Worker_SRC_Files) $(Client_SRC_Files)
 	@echo
-	@echo "Building the integritee-service"
-	@cd service && SGX_SDK=$(SGX_SDK) SGX_MODE=$(SGX_MODE) cargo build $(Worker_Rust_Flags)
+	@echo "Building the integritee-service and integritee-cli"
```

Comment:
Have you tried if we get the same speed-up if we still separate the two, but with:

* `cargo build -p integritee-cli`
* `cargo build -p integritee-service`
?

Reply:
I have not, no - what benefit would that have?

Reply:
I would still be able to build them separately.

Reply:
Seeing your commit, I believe that I have not expressed myself well. I wanted to still have them exposed as two different makefile commands to build them separately. But then again, it does not matter to me too much. :)

Reply:
Ah okay, I understand now 😄 it's certainly worth a try 👍